src/components/Layout.tsx

```diff
@@ -37,10 +32,6 @@ import FortressLogo from "~/svg/fortress-logo.svg"
 
 const Layout: FC<PropsWithChildren> = ({ children }) => {
   const [sidebarOpen, setSidebarOpen] = useState(false)
-  const [connectModal, setConnectModal] = useConnectWallet((state) => [
```

Comment:
Best to not have state changing this high up in the component tree when it can be moved into into isolated components.